src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppInsights.java

```diff
@@ -43,6 +53,32 @@ public void trackFtpUpload(java.time.Duration duration, boolean success) {
         );
     }
 
+    private double getSecondsSince(Timestamp time) {
+        if (time == null) {
+            return 0;
+        }
+
+        long seconds = SECONDS.between(time.toInstant(), Instant.now());
+
+        return ((double) seconds) / SECONDS_PER_DAY;
+    }
+
+    public void trackNotPrintedLetter(StaleLetter staleLetter) {
+        telemetry.trackEvent(
+            LETTER_NOT_PRINTED,
+            ImmutableMap.of(
+                "letterId", staleLetter.getId().toString(),
+                "messageId", staleLetter.getMessageId(),
+                "service", staleLetter.getService(),
+                "type", staleLetter.getType()
+            ),
+            ImmutableMap.of(
+                "daysCreated", getSecondsSince(staleLetter.getCreatedAt()),
+                "daysSentToPrint", getSecondsSince(staleLetter.getSentToPrintAt())
```

Comment:
this is tightly related to when the job runs. perhaps we could store information on what time of the day letter was create and what day of the week instead.